projects/plugins/jetpack/extensions/blocks/subscriptions/test/fixtures/jetpack__subscriptions.html

```diff
@@ -1,3 +1,3 @@
-<!-- wp:jetpack/subscriptions {"showSubscribersTotal":true,"buttonOnNewLine":true,"fontSize":33,"customFontSize":33,"borderRadius":8,"borderWeight":3,"padding":18} -->
-<div class="wp-block-jetpack-subscriptions wp-block-jetpack-subscriptions__supports-newline wp-block-jetpack-subscriptions__use-newline wp-block-jetpack-subscriptions__show-subs">[jetpack_subscription_form show_subscribers_total="true" button_on_newline="true" custom_font_size="33" custom_border_radius="8" custom_border_weight="3" custom_padding="18" custom_spacing="10" submit_button_classes="has-33-font-size" email_field_classes="has-33-font-size" show_only_email_and_button="true"]</div>
+<!-- wp:jetpack/subscriptions {"subscribePlaceholder":"Enter your email address","showSubscribersTotal":true,"buttonOnNewLine":true,"submitButtonText":"Sign Up","fontSize":"33px","customFontSize":"33px","borderRadius":8,"borderWeight":3,"padding":18,"successMessage":"Volla, you subscribed to our blog!"} -->
```

Comment:
Were these non-v6 files automatically generated for you? I tried generating the fixtures on my end but it only produced `deprecated-6` items. I'm thinking once the `save.js` is fixed up, you might try deleting the fixtures here and generating theme once more. Here is what my starting .html v6 looked like:

```html
<!-- wp:jetpack/subscriptions {"successMessage":"Custom success message text."} -->
<div class="wp-block-jetpack-subscriptions wp-block-jetpack-subscriptions__supports-newline">[jetpack_subscription_form show_subscribers_total="false" button_on_newline="false" custom_font_size="16px" custom_border_radius="0" custom_border_weight="1" custom_padding="15" custom_spacing="10" submit_button_classes="" email_field_classes="" show_only_email_and_button="true" success_message="Custom success message text."]</div>
<!-- /wp:jetpack/subscriptions -->
```

Reply:
Non-v6 files were not generated in my case. It is only generating/modifying files for v6.